packages/api/src/controllers/stream.ts

```diff
@@ -401,6 +401,31 @@ app.get("/", authorizer({}), async (req, res) => {
   );
 });
 
+export async function getRecordingPlaybackUrl(
+  stream: DBStream,
+  objectStoreId: string
+) {
+  let url: string;
+
+  try {
+    const session = await db.stream.getLastSessionFromSessionsTable(stream.id);
+
+    if (!session) {
```

Comment:
Should we check `session.record`? (or `stream.record`)

Reply:
We're checking it up in the code, on the api call